src/main/java/com/hida/DatabaseManager.java

```diff
@@ -0,0 +1,261 @@
+     */
+    public synchronized void addId(Set<String> list) {
+        //ArrayList<String> redundantIdList = new ArrayList();
+        try {
+
+            // a string to hold sqlite3 queries
+            String sqlQuery;
+
+            for (String id : list) {
+                // assign a sqlite3 query that adds an id to the database
+                System.out.println("adding id...");
+                sqlQuery = String.format("INSERT INTO %s (%s) "
+                        + "VALUES('%s')", TABLE_NAME, COLUMN_NAME , id);
+
+                // a statement that allows database/webservice communication
+                Statement insertStatement
+                        = DATABASE_CONNECTION.createStatement();
+
+                // execute query
+                insertStatement.executeUpdate(sqlQuery);
+                System.out.println("id added");
+
+                // clean-up                
+                insertStatement.close();
+            }
+
+        } catch (SQLException exception) {
+            System.err.println(exception.getMessage() + " in addId");
+            System.err.println(Arrays.toString(exception.getStackTrace()));
+        }
+        System.out.println("printed to database");
+        // prints out to server what ids were created
+        //printData();
+        
+    }
+
+    /**
+     * Checks a list of ids against a database to ensure that no redundancies
+     * are added.
+     *
+     * @param list - list of ids to check.
+     * @return - a list containing ids that already exist in database. Returns
+     * an empty list of all the ids given in the param list are unique.
+     */
+    public Set<String> checkId(Set<String> list) {
+        // a list containing redundant ids
+        Set<String> redundantIdList = new HashSet();
+        try {
+
+            // a string to query a specific id
+            String sqlQuery;
+
+            for (String id : list) {
+
+                // a string to query a specific id for existence in database
+                sqlQuery = String.format("SELECT %s FROM %2$s WHERE "
+                        + "%1$s = '%3$s'", COLUMN_NAME, TABLE_NAME, id);
+
+                // a statement that allows database/webservice communication
+                Statement databaseStatement
+                        = DATABASE_CONNECTION.createStatement();
+
+                // execute statement and retrieves the result
+                ResultSet databaseResponse
+                        = databaseStatement.executeQuery(sqlQuery);
+
+                // adds id to redundantIdList if it already exists in database
+                if (databaseResponse.next()) {
+                    System.out.print("redundancy list: " + id);
+                    redundantIdList.add(id);
+                }
+
+                // clean-up                
+                databaseResponse.close();
+                databaseStatement.close();
+
+            }
+
+        } catch (SQLException exception) {

```

Comment:
In general you shouldn't just ignore exceptions like this because they signal that something went wrong.  Unless you're actually going to handle it, log it (and rethrow), or wrap the exception in another exception, it's better if you don't have a `try-catch` here